src/main.js

```diff
@@ -55,8 +55,11 @@ if (window.location.search.indexOf("testEnvironment") > -1) {
      * navigator.language in ExtensionLoader (when making require contexts for each
      * extension).
      */
+    if(window.location.search) {
+        var locale = /locale=([^&]+)&?/.exec(window.location.search)[1] || "en";
```

Comment:
Hmm, even that isn't a great solution since the defaults will be lost if `search` exists (which it should).